packages/gateway/src/features/dataSeeding/certificateSeeding.ts

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable prettier/prettier */
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+
+import { COUNTRY_CONFIG_URL, APPLICATION_CONFIG_URL } from '@gateway/constants'
+
+import { GQLCertificateStatus, GQLEvent } from '@gateway/graphql/schema'
+
+import fetch from 'node-fetch'
+
+import { hasScope } from '@gateway/features/user/utils'
+import { uploadSvg } from '@gateway/utils/documents'
+
+interface IcertificateMeta {
+  event: GQLEvent
+  fileName: string
+  svgCode: string
+}
+
+async function getCertificate() {
+  const url = new URL('certificates', COUNTRY_CONFIG_URL).toString()
+  const res = await fetch(url)
+  if (!res.ok) {
+    throw new Error(`Expected to get the certificates from ${url}`)
+  }
+  return res.json()
+}
+
+async function uploadCertificate(token: string, certificate: IcertificateMeta) {
+  const authHeader = {
+    Authorization: `Bearer ${token}`
+  }
+  const certificateSVG = {
+    svgCode: certificate.svgCode as string,
+    svgFilename: certificate.fileName,
+    user: 'jonathan.campbell',
+    event: certificate.event,
+    status: GQLCertificateStatus.ACTIVE
+  }
+
+  if (!hasScope(authHeader, 'natlsysadmin')) {
+    return await Promise.reject(
+      new Error('Create or update certificate is only allowed for natlsysadmin')
+    )
+  }
```

Comment:
No need to check for scope as we know we are only doing this with our super user